### ISSUE TITLE
Improper Pagination Markup

### ISSUE THREAD

**Issue by ScriptEngineer** [Mobius1/Vanilla-DataTables#69]
The default-rendered pagination creates _li_ elements inside a _div_ instead of the proper _ul_ or _ol_ tags.

**Comment by leejordan** [comment]
Yes this stands out as a big problem. Is this repo still being worked on though? For my money it would be good to be able to fix a lot of the markup that I find unsatisfying such as no label on the search input and other bits.
